cherrypy/process/servers.py

```diff
@@ -185,7 +185,7 @@ def _get_base(self):
         if not self.httpserver:
             return ''
         host, port = self.bind_addr
-        if getattr(self.httpserver, 'ssl_adapter', None):
+        if getattr(self,'ssl_certificate', None):
```

Comment:
Only the test with Python:nightly build ( python 3.7 ) is failing. Tests with all other versions give positive results. How do I fix this?